src/corecel/sys/Stream.cc

```diff
@@ -35,9 +36,11 @@ auto AsyncMemoryResource<Pointer>::do_allocate(
  * Deallocate device memory.
  */
 template<class Pointer>
-void AsyncMemoryResource<Pointer>::do_deallocate(
-    CELER_UNUSED_UNLESS_DEVICE pointer p, std::size_t, std::size_t)
+void AsyncMemoryResource<Pointer>::do_deallocate(pointer p,
```

Comment:
[pre-existing code question]  Why does the `do_deallocate` function takes 2 sizes as argument?

Reply:
Should be posted on #908 😉  This is to accommodate a thrust API. I'm guessing it's an alignment requirement, which isn't needed here because CUDA always aligns to something absurdly large (256 bytes IIRC).